### PR TITLE
Update cli.py to grant access to multiple sensors

### PR DESCRIPTION
**Pull request by foundbobby**
Granting access to multiple sensors. Tested with TemperNTC, sample output below:

python cli.py
Found 1 devices
Device #0: 11.3°C 52.4°F

python cli.py -s 0
Found 1 devices
Device #0: 11.3°C 52.4°F

python cli.py -s 1
Found 1 devices
Device #0: 8.1°C 46.5°F

python cli.py -q -c -s 0
11.3

python cli.py -q -c -s 1
8.1

### CODE REVIEW

File: temperusb/cli.py [hunk 1]
```diff
@@ -29,6 +31,11 @@ def main():
             degree_unit = 'f'
         elif o == "-q":
             quiet_output = True
+        elif o == "-s":
+            try:
+                sensor_id = int(float(a))
```
Comment (ps-jay) [inline temperusb/cli.py]: Why cast to float first?

**Comment by ps-jay** [comment]
Thanks for the contribution : )

File: temperusb/cli.py (hunk 1 above)
Comment (foundbobby) [inline temperusb/cli.py]: Cleaned up, no reason to cast to float first.

**Comment by foundbobby** [comment]
Updated to remove float cast

**Pull request merged**